mart/models/modular.py

```diff
@@ -80,6 +79,14 @@ def __init__(
         self.test_step_log = test_step_log or []
         self.test_metrics = test_metrics
 
+        # Load state dict for specified modules
+        load_state_dict = load_state_dict or {}
+        # Hydra commandlines converts dotted path to nested dictionary.
+        for name, path in get_dottedpath_items(load_state_dict):
+            module = attrgetter(name)(self.model)
+            logger.info(f"Loading state_dict {path} for {module.__class__.__name__}...")
+            module.load_state_dict(torch.load(path))
```

Comment:
Shall we add `map_location="cpu"` in case no GPU is available?

Reply:
I guess that is what is in the original implementation so sure.